airflow/contrib/secrets/__init__.py

```diff
@@ -0,0 +1,26 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This package is deprecated. Please use `airflow.secrets` or `airflow.providers.*.secrets`."""
+
+import warnings
+
+warnings.warn(
+    "This package is deprecated. Please use `airflow.secrets` or `airflow.providers.*.secrets`.",
+    DeprecationWarning,
+    stacklevel=2,
+)
```

Comment:
```suggestion
```
I would probably say no deprecation is needed here -- this will issue two deprecations as a result, cos importing any package under this will have to import this too.